bob-access/Cargo.toml

```diff
@@ -9,12 +9,12 @@ axum = "0.4"
 bitflags = "1.3"
 hex = "0.4.3"
 http = "0.2"
-sha2 = "0.10.2"
+sha2 = "0.10.6"
 tower = "0.4"
 tonic = "0.6"
 log = "0.4"
 serde = {version = "1.0", features = ["derive"] }
 jsonwebtoken = "7.2"
-serde_yaml = "0.8"
+serde_yaml = "0.9"
```

Comment:
You wrote that using version `0.9` causes panic. Why did you update it?

Reply:
Only in bob-common.

Reply:
I have unified `serde_yaml` version.